--- a/setup.py
+++ b/setup.py
@@ -47,6 +47,7 @@
 
 
 def download_library(command):
+    return
     if command.dry_run:
         return
     libdir = absolute('libsecp256k1')
@@ -269,13 +270,7 @@
         # ABI?: py_limited_api=True,
     )
 
-    extension.extra_compile_args = [
-        subprocess.check_output(['pkg-config', '--cflags-only-I', 'libsecp256k1']).strip().decode('utf-8')  # noqa S603
-    ]
-    extension.extra_link_args = [
-        subprocess.check_output(['pkg-config', '--libs-only-L', 'libsecp256k1']).strip().decode('utf-8'),  # noqa S603
-        subprocess.check_output(['pkg-config', '--libs-only-l', 'libsecp256k1']).strip().decode('utf-8'),  # noqa S603
-    ]
+    extension.extra_link_args = ['-lsecp256k1']
 
     if os.name == 'nt' or sys.platform == 'win32':
         # Apparently, the linker on Windows interprets -lxxx as xxx.lib, not libxxx.lib
@@ -340,7 +335,7 @@
     license='MIT OR Apache-2.0',
 
     python_requires='>=3.8',
-    install_requires=['asn1crypto', 'cffi>=1.3.0'],
+    install_requires=[],
 
     packages=find_packages(exclude=('_cffi_build', '_cffi_build.*', 'libsecp256k1', 'tests')),
     package_data=package_data,
--- a/setup_support.py
+++ b/setup_support.py
@@ -56,6 +56,7 @@
 
 
 def _find_lib():
+    return True
     if 'COINCURVE_IGNORE_SYSTEM_LIB' in os.environ:
         return False
 
